### PR TITLE
remove get_dirs and cache in get_attr

### PR DESCRIPTION
**Pull request by poupotte**
Test : 
-  remove get_dirs
-  remove cache in get_attr

### CODE REVIEW

File: cozyfuse/couchmount.py [hunk 1]
```diff
@@ -34,13 +34,19 @@
 logger.setLevel(logging.INFO)
 #local_config.configure_logger(logger)
 
+def get_current_date():
+    return datetime.datetime.now().strftime('%Y-%m-%dT%H:%M:%S')
```
Comment (frankrousseau) [inline cozyfuse/couchmount.py:42]: Could you add a comment explaining what does the function?

File: cozyfuse/couchmount.py [hunk 2]
```diff
@@ -613,6 +541,12 @@ def _replicate_from_local(self, ids):
             doc_ids=ids
         )
 
+    def _update_parent_folder(self, parent_folder):
```
Comment (frankrousseau) [inline cozyfuse/couchmount.py:536]: Comment is missing here too.

**Comment by frankrousseau** [comment]
Thank you @poupotte for the PR.

So, you keep cache for file descriptors ?

**Comment by poupotte** [comment]
No, I don't keep cache for file descriptors in this version.
But maybe the best solution is : 
- Remove get_dirs
- Keep get_attr (file descriptor) in cache

However, if we decide to keep file descriptor, we have to manage updating with synchronisation.

**Pull request merged**